kw

```diff
@@ -36,6 +36,18 @@ KW_STATE_DIR="${XDG_STATE_HOME:-"${HOME}/.local/state"}/${KWORKFLOW}"
 # ENV dir name
 ENV_DIR='envs'
 
+# Export external variables required by kworkflow
+export KWORKFLOW
+
+#INJECT_CODE_TRACING_SETUP
+
+# This is the one and only time a file will be sourced this way.
+# The include function (sourced from this file) should always be used for file sourcing.
+. "${KW_LIB_DIR}/lib/kw_include.sh" --source-only
+
+# Dev mode has to begin after sourcing lib/kw_include.sh
+# Otherwise, it is not possible to use helpers.
```

Comment:
I see that you moved this sourcing to be able to use the `say` function, but moving it to here introduced a regression. When I run kw using the repo executable without having kw installed in my system, kw crashes:

![Screen Shot 2023-11-17 at 22 10 18](https://github.com/kworkflow/kworkflow/assets/95055784/bcb437e2-e7a4-4e69-9d7e-1632c722a0ae)

I think that this is due to the else in the if-else below being executed at the start of the kw file:

```bash
# Kw shell files
if [[ -f '/usr/bin/kw' ]]; then
  KW_LIB_DIR='/usr/share/kw'
else
  KW_LIB_DIR="${KW_LIB_DIR:-"${HOME}/.local/lib"}/${KWORKFLOW}"
fi
```

You can either drop this move and don't use the `say` function (however we lose colors 😢), or source the file directly from the repo without using `include` (but I don't think breaking the rule of sourcing with `include` is wise). You could also use colors directly, but you need to be extra sure to not mess the user terminal session 🙃.

Reply:
I won't use the `say` function and will instead use some colors directly